flexmeasures/ui/templates/crud/account.html

```diff
@@ -1,183 +1,514 @@
+                  >
+                </td>
+                <td>
+                  {% if asset.latitude and asset.longitude %} LAT: {{
+                  "{:,.4f}".format( asset.latitude ) }} LONG: {{
+                  "{:,.4f}".format( asset.longitude ) }} {% endif %}
+                </td>
+                <td>{{ asset.id }}</td>
+                <td>
+                  {% if asset.owner %} {{ asset.owner.name }} {% else %} PUBLIC
+                  {% endif %}
+                </td>
+                <td>{{ asset.sensors | length }}</td>
+                <td>
+                  <a href="/assets/{{ asset.id }}/status">
+                    <button type="button" class="btn">Status</button>
+                  </a>
+                </td>
+                <td class="text-right"></td>
+                <td class="d-none">/assets/{{ asset.id }}</td>
+              </tr>
+              {% endfor %}
+            </tbody>
+          </table>
+        </div>
+      </div>
+    </div>
+    <div class="col-md-2"></div>
+  </div>
 </div>
-<div class="col-md-2"></div>
 {% block paginate_tables_script %} {{ super() }} {% endblock %}
+<script type="text/javascript">
+
+  const basePath = window.location.origin;
+  const form = document.getElementById("editaccount");
+  const toastStack = document.getElementById("toast-container");
+  const closeToastBtn = document.getElementById("close-all-toasts");
+
+  // hide button
+  closeToastBtn.style.display = "none";
+
+  closeToastBtn.addEventListener("click", function () {
+    const toastElements = document.querySelectorAll(".toast");
+    toastElements.forEach((toast) => {
+      const toastInstance = bootstrap.Toast.getInstance(toast); // Get the toast instance
+      if (toastInstance) {
+        // destroy the toast
+        toastInstance.dispose();
+        toast.remove();
+      }
+    });
+    // Hide the close button
+    closeToastBtn.style.display = "none";
+  });
+
+  function showToast(message, type) {
+    let colorClass;
+    let colorStyle;
+    let title;
+    if (type == "error") {
+      colorClass = "bg-danger";
+      title = "Error";
+    } else if (type == "success") {
+      colorClass = "bg-success";
+      title = "Success";
+    } else {
+      colorStyle =
+        "background-color: {{ account.primary_color or primary_color }};";
+      title = "Info";
+    }
+    const toast = `
+        <div class="toast hide mb-1" data-bs-autohide="true" role="alert" aria-live="assertive" aria-atomic="true">
+          <div class="toast-header">
+            <div
+            class="rounded me-2 ${colorClass}"
+                  style="
+                    width: 20px;
+                    height: 20px;
+                    display: inline-block;
+                    ${colorStyle}
+                  "
+                ></div>
+            <strong class="me-auto">${title}</strong>
+            <button type="button" class="btn-close" data-bs-dismiss="toast" aria-label="Close"></button>
```

Comment:
This "dismisses" the toast, but it will re-appear if I make another mistake in the form.
I guess I want the same to happen here as with the button that closes all toasts. The element should be gone.